app/code/community/Bolt/Boltpay/Block/Checkout/Boltpay.php

```diff
@@ -135,8 +135,13 @@ protected function isAdminAndUseJsInAdmin($checkoutType)
      */
     public function buildCartData($orderCreationResponse, $checkoutType = null)
     {
-        $authCapture = $checkoutType === 'admin' ? true : Mage::getStoreConfigFlag('payment/boltpay/auto_capture');
-
+        if($checkoutType === 'admin') {
```

Comment:
This is good but we have so many `extraConfig` settings. I'm going to get Mitch's opinion here on to how to best document these externally.